js/apps/account-ui/src/resources/Resources.tsx

```diff
@@ -18,12 +18,14 @@ const Resources = () => {
         unmountOnExit
       >
         <Tab
+          data-testId="myResources"
```

Comment:
Haven't seen this casing convention before. Can we keep it the consistent with the other test ids? (same goes for the others in this PR).
```suggestion
          data-testid="myResources"
```